W-22671160-Add-examples-for-MS-Vibes-CP (#653)
mulesoft/docs-code-builder: modules/ROOT/pages/vibes-workflows-commands.adoc

:page-aliases: a4d-workflows-commands.adoc

> Workflows and rules let you customize how MuleSoft Vibes operates within your project.

Reviewer: suggested by Notebook to avoid "let you"

```suggestion
Use workflows and rules to customize how MuleSoft Vibes operates within your project.
```

Author: OK! Thanks!

* *API creation workflow*: Generate an API specification, apply validation rules, suggest improvements, and save the final version to the workspace.

* *Troubleshooting workflow*: Analyze an error, identify the root cause, propose a fix, and update the configuration if needed.

Example workflow definition to enter in *Add Workflow*:

[source,text]
----
Name: api-design-review
Description: Run governance checks before publishing a RAML spec.
Steps:
1. Validate RAML syntax and required metadata.
2. Check naming conventions from workspace rules.
3. Check for unsecured endpoints that expose sensitive fields.
4. Summarize findings with required fixes and suggested improvements.
----

[[add-commands]]
== Add Commands

Commands provide quick shortcuts for interacting with MuleSoft Vibes.  
You can invoke any command by typing `/` in the message input box.

MuleSoft Vibes includes several built-in commands.

[cols="1,3",options="header"]
|===
| Command | Description

| `/newrule`
| Create a new rule based on the current conversation.

| `/newchat`
| Start a new chat that carries over context from your current task.

| `/reportbug`

> | Report an issue by creating a GitHub ticket with Agentforce Vibes formatting.

Reviewer: not a big one, but do we say "ticket" or is this a bug or GitHub issue or...?

Author: Great catch! Let's see if this rewording works.
